HACKING.org

```diff
@@ -1,5 +1,21 @@
+* Testing on other projects
+
+The =tools/test_branch.sh= script builds 2 revisions of OCamlformat and run them
+on the sources of some projects.
+The list of tested project is (and can be edited) in =test-extra/Makefile=.
+
+The projects sources are cloned into =test-extra/code=.
+Git is used to manage the script's state:
+- staged changes are the diffs introduced by formatting with the first revision
+- unstaged changes are the diffs introduced by upgrading OCamlformat to the
+  second rev
+
 * Release procedure
 
+- Review the impact of the release on real source code:
+  In =test-extra/Makefile=, uncomment the extended list of projects to test
+  and run =tools/test_branch.sh -a "previous_release"=.
```

Comment:
It's *very* important that this review is made externally visible to the maintainers of these projects and that a discussion is started with them to decide whether the changes are acceptable. 

Please add an item to reflect this in the release procedure.

Reply:
I clarified that